.github/workflows/ci.yml

```diff
@@ -223,3 +229,47 @@ jobs:
           name: "Log files Kanto"
           path: |
             runtime_kanto/test/logs
+
+  run-desired-state-generator-tests:
+    name: Run Integration Tests for desired state generator
```

Comment:
I think it would be a good idea to have an own workflow file for running the integration tests. We are now repeating almost everything 4 times beside some small parts. What do you think?

Reply:
extracted some stuff of it